common/supplemental/supplementalData.xml

```diff
@@ -3696,6 +3696,7 @@ XXX Code for transations where no currency is involved
 			<languagePopulation type="tr" populationPercent="1.2"/>	<!--Turkish-->
 		</territory>
 		<territory type="NO" gdp="381200000000" literacyPercent="100" population="5467440">	<!--Norway-->
+			<languagePopulation type="no" populationPercent="100" officialStatus="official"/>	<!--Norwegian -->
```

Comment:
We will want to add a line for this in country_language_population_raw.txt as well (otherwise this will get overwritten), but I can do that as part of other changes I plan to make today.